eppo_core/src/poller_thread.rs

```diff
@@ -79,7 +143,7 @@ impl PollerThread {
                             }
                         };
 
-                        let timeout = jitter(POLL_INTERVAL, POLL_JITTER);
+                        let timeout = config.interval + jitter(config.jitter);
```

Comment:
Nit: we should subtract the jitter to make sure we always poll within the interval as set up (the interval is an upper bound instead of a lower bound)

Reply:
Note: will have to make sure that jitter < interval ( / 2?)

Reply:
Good catch! I somehow missed that jitter should be subtracted during initial implementation

I have fixed that and added two tests: to check that jitter is subtractive + that underflow (`interval - jitter < 0`) is handled correctly